packages/browser/src/utils/lazyLoadIntegration.ts

```diff
@@ -33,12 +33,15 @@ const WindowWithMaybeIntegration = WINDOW as {
 export async function lazyLoadIntegration(name: keyof typeof LazyLoadableIntegrations): Promise<IntegrationFn> {
   const bundle = LazyLoadableIntegrations[name];
 
-  if (!bundle || !WindowWithMaybeIntegration.Sentry) {
+  // `window.Sentry` is only set when using a CDN bundle, but this method can also be used via the NPM package
+  const sentryOnWindow = (WindowWithMaybeIntegration.Sentry = WindowWithMaybeIntegration.Sentry || {});
+
+  if (!bundle) {
     throw new Error(`Cannot lazy load integration: ${name}`);
   }
 
   // Bail if the integration already exists
-  const existing = WindowWithMaybeIntegration.Sentry[name];
+  const existing = sentryOnWindow[name];
```

Comment:
related to my comment above: Assuming we already installed the integration in NPM mode during the normal `Sentry.init` call, it wouldn't have been registered on `window.Sentry` so we wouldn't actually bail here. 

Again, I feel like I lack some context so feel free to disregard!

Reply:
So this case specifically I thought about too, my idea was that we can/should check for this outside of this method - my reasoning was that otherwise we'd have to _also_ pass the integration name (e.g. `HttpClient` ) to the function because we'd need this to lookup via `getIntegrationByName()` 🤔 No strong feelings, we can also extend the function signature and check in here?

Reply:
cc @ryan953 also relates to your comment. So to clarify we have to options IMHO:

1. Keep this function "lean" so it does not check integration name - this will keep the bundle size slightly smaller, and instead at call site you have to do something like this:

```js
function maybeLoadFeedback() {
  if(client.getIntegrationByName('Feedback')) {
    return;
  }
  return lazyLoadIntegration('feedbackIntegration');
}
```

2. Do the check in the integration: For this we have to options, either requiring to pass in both function name as well as integration name, or adding this to the map somehow. The latter has a nicer DX, but will increase bundle size:

2.a: `lazyLoadIntegration('feedbackIntegration', 'Feedback')` <-- smaller bundle size, but not-so-nice usage
2.b: `lazyLoadIntegration('feedbackIntegration')` and keep `feedbackIntegration > Feedback` in the map somehow

For me, the main reason I went with 1. was that if a user uses this, they probably don't need this check (think in the future we want users to use this when using the loader to add pluggable integrations), and thus we can keep the API surface & bundle size slightly smaller. But I'm also totally fine with adding this additional check.

Reply:
I like #1 for what it's worth.
In the case of an npm bundle the user could easily have `feedbackIntegration` or something included in the bundle, but not added at runtime, therefore `getIntegrationByName` would not return and we'd still inject a script tag to load the integration again. For this reason I think it makes sense to avoid the extra check inside the method.

Reply:
Ok yes, that makes sense, let's go with 1 then. Thx for explaining!